th2_data_services/interfaces/adapter.py

```diff
@@ -31,15 +47,15 @@ def handle_stream(self, stream: Iterable):
         pass
 
 
-class IMessageAdapter(IAdapter):
+class IMessageAdapter(IRecordAdapter):
     """Interface of Adapter for messages."""
 
     @abstractmethod
     def handle(self, message: dict) -> Any:
         pass
 
 
-class IEventAdapter(IAdapter):
+class IEventAdapter(IRecordAdapter):
     """Interface of Adapter for events."""
 
     @abstractmethod
```

Comment:
Perhaps we don't need these classes anymore.
Analyze, please.

Reply:
I searched through the project and IMessageAdapter is never used, nor IEventAdapter. Remove them?

Reply:
Let's remove in this case.